content/applications/sales/crm/track_leads/lead_distribution_report.rst

```diff
@@ -0,0 +1,218 @@
+
+#. Lead creation date
+#. Sales team location
+#. Contains a phone number
+#. An active state is set
+#. Referred by appointment
+#. Source is livechat
+#. Notes are set
+#. Tagged with a size from 10-20 or 20-50
+#. Email does not contain bad addresses
+#. An active salesperson is set
+
+Any number of conditions can be added, removed, or modified to best fit the desired information in
+the report.
+
+Add lead creation date
+~~~~~~~~~~~~~~~~~~~~~~
+
+Click the first field on the left under :guilabel:`Match any of the following rules:` with the value
+:guilabel:`Country` in it. Type :guilabel:`Date` in the :guilabel:`Search...` bar, or scroll to
+search through the list to locate it. Then, click the next field and select :guilabel:`>=` from the
+drop-down menu. This operator will include only values greater than or equal to the value in the
+rightmost field.
+
+The rightmost field should contain the earliest date leads are selected from. For example,
+setting `01/01/2024 00:00:05` will include only leads created from, and including, the first day of
+2024.
+
+.. image:: lead_distribution_report/created-on.png
+   :align: center
+   :alt: Add a Created On rule for the start of the year onward.
+
+Add sales team location
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Click :guilabel:`New rule` to add a new rule and another horizontal set of fields will appear. Click
+the first field for the new rule, and set it to :guilabel:`Sales Team`. Then, click the next
+field and select :guilabel:`contains` from the drop-down menu. Selecting this operator will filter
+for any records that contain the words in the rightmost field.
+
+In this third field, enter the name of the desired sales teams. For example, setting `us direct sf
+northam` will include only sales teams assigned to the U.S., direct, San Francisco, and North
+America.
+
+.. image:: lead_distribution_report/sales-team-location.png
+   :align: center
+   :alt: Use Sales Team to filter the location the lead is associated with.
+
+Add phone number
+~~~~~~~~~~~~~~~~
+
+Click :guilabel:`New rule` to add a new rule to the filter. Set the first field to
+:guilabel:`Phone`. Then, click the next field and select :guilabel:`is set` from the drop-down menu.
+Selecting this operator will filter only for records that have a phone number associated with the
+lead.
+
+.. image:: lead_distribution_report/phone-set.png
+   :align: center
+   :alt: Report only leads with an associated phone number with the Phone field.
+
+Add active status
+~~~~~~~~~~~~~~~~~
+
+Click the :guilabel:`Branch` icon to the right of :guilabel:`Phone is set` to add a new rule that
+branches from the rules above.
+
+Two horizontal sets of fields will appear below a line showing
+:guilabel:`all of:` with a :icon:`fa-caret-down` down arrow. Click the down arrow, then select
+:guilabel:`any` from the list.
+
+Set the first field to :guilabel:`Active`. Then, click the rightmost field and select
+:guilabel:`is set` from the drop-down menu.
```

Comment:
![Screenshot 2024-06-24 130604](https://github.com/odoo/documentation/assets/73958186/2b5e0b1e-4e33-4254-822e-6324941746cd)

During my run-through, I had to keep an option selected in the second and third fields...there wasn't just an "is set" choice in the third field... might be worth a quick revisit/rewrite of this portion

Reply:
UPDATE: so, I've been using a 17.0 runbot for this review, because that's what this PR is listed as targeting... however, I just tried out this flow in the 17.2 runbot, and there *is* an accessible option like this in the second field.